test/e2e/data/shared/v1beta1_provider/metadata.yaml

```diff
@@ -19,3 +19,6 @@ releaseSeries:
   - major: 1
```

Comment:
If we want to stay extra tidy, we should be able to remove the 1.11 and 1.12 stanzas here.

Reply:
@nawazkh could you do that as a follow-up PR once this lands?

Lemme know if you don't understand why we would do this, but tl;dr we're maintaining a static set of CAPZ version that we test in azure-dev.yaml, and we only need entries in this file for the versions that we have test configurations to actually use.